crates/orchestrator/src/network.rs

```diff
@@ -668,8 +691,10 @@ impl<T: FileSystem> Network<T> {
             self.relay.nodes.push(node.clone());
         }
         // TODO: we should hold a ref to the node in the vec in the future.
+        node.mark_running();
```

Comment:
this fn is one line, and only used here. Make sense to just inline the call here?

Reply:
ahh oks, then we can keep it. thx!